build.sbt

```diff
@@ -22,6 +23,12 @@ def common: Seq[Setting[_]] =
     Compile / javacOptions ++= Seq("-encoding", "UTF-8", "--release", "11"),
     Compile / scalacOptions ++= Seq("-encoding", "UTF-8", "-release", "11"))
 
+lazy val codegenProtoAnnotations = project
+  .in(file("codegen/proto-annotations"))
+  .enablePlugins(PublicProtocolProject)
+  .settings(common)
+  .settings(name := "akkaserverless-codegen-proto-annotations")
```

Comment:
will these be part of the java-sdk or is this only temporary?

Reply:
We have been discussing it. Johan made the point that since it's a codegen concern, we could keep it in the codegen itself and make it part of the SDK. There are pros and cons about it and I don't think the decision is settled. 

We are keeping it here for now to move fast and have more time to think about it.